specification.json

```diff
@@ -59,32 +59,32 @@
         {
             "id": "Requirement 1.3.1",
             "machine_id": "requirement_1_3_1",
-            "content": "The `client` MUST provide methods for flag evaluation, with parameters `flag key` (string, required), `default value` (boolean | number | string | structure, required), `evaluation context` (optional), and `evaluation options` (optional), which returns the flag value.",
+            "content": "The `client` MUST provide methods for typed flag evaluation, including boolean, numeric, string, and structure, with parameters `flag key` (string, required), `default value` (boolean | number | string | structure, required), `evaluation context` (optional), and `evaluation options` (optional), which returns the flag value.",
             "RFC 2119 keyword": "MUST",
             "children": []
         },
         {
             "id": "Condition 1.3.2",
             "machine_id": "condition_1_3_2",
-            "content": "The language type system differentiates between strings, numbers, booleans and structures.",
+            "content": "The implementation language differentiate between floats/ints.",
             "RFC 2119 keyword": null,
             "children": [
                 {
                     "id": "Conditional Requirement 1.3.2.1",
                     "machine_id": "conditional_requirement_1_3_2_1",
-                    "content": "The `client` MUST provide methods for typed flag evaluation, including boolean, numeric, string, and structure.",
-                    "RFC 2119 keyword": "MUST",
-                    "children": []
-                },
-                {
-                    "id": "Conditional Requirement 1.3.3",
-                    "machine_id": "conditional_requirement_1_3_3",
-                    "content": "The `client` SHOULD guarantee the returned value of any typed flag evaluation method is of the expected type. If the value returned by the underlying provider implementation does not match the expected type, it's to be considered abnormal execution, and the supplied `default value` should be returned.",
+                    "content": "The client SHOULD provide functions for all types, consistent with language idioms.",
```

Comment:
I think we should link to the "types" doc here if we say "all types"... but it might be better to say "strings, numbers, booleans and structures"

Reply:
I have made this change, so and we have added a link to types for more details.